source/index.txt

```diff
@@ -1,80 +1,247 @@
+            in the Atlas documentation.
+
+            For details about connecting to a non-default local port, 
+            connecting with authentication, or other options, refer to 
+            :ref:`mdb-shell-connect`.
+
+      .. image:: /images/mongosh-plp/atlas_plp.gif
+         :alt: MongoDB Atlas Connect to your Database image
+
+   .. tab:: Perform CRUD and Aggregation
+      :tabid: crud
+
+      .. procedure::
+
+         .. step:: Insert Documents
+
+            :binary:`~bin.mongosh` supports common insert opererations, 
+            including:
+
+            - :method:`db.collection.insertOne()`
+            - :method:`db.collection.insertMany()`
+
+            For more information and examples, refer to :ref:`mongosh-insert`.
+
+         .. step:: Read Documents
+
+            Use the :method:`db.collection.find()` method to query documents
+            in a collection. For more information and examples, refer to 
+            :ref:`mongosh-read`.
+
+         .. step:: Update Documents
+
+            :binary:`~bin.mongosh` supports common update opererations, 
+            including:
+
+            - :method:`db.collection.updateOne()`
+            - :method:`db.collection.updateMany()`
+            - :method:`db.collection.replaceOne()`
+
+            For more information and examples, refer to :ref:`mongosh-update`.
+
+         .. step:: Delete Documents
+
+            :binary:`~bin.mongosh` supports common delete opererations, 
+            including:
+
+            - :method:`db.collection.deleteMany()`
+            - :method:`db.collection.deleteOne()`
+
+            For more information and examples, refer to :ref:`mongosh-delete`.
+
+         .. step:: Run Aggregation Pipelines
+
+            You can run aggregation pipelines in :binary:`~bin.mongosh` 
+            using the :method:`db.collection.aggregate()` method. For more 
+            information and examples, refer to :ref:`mdb-shell-aggregation`.
+
+      .. image:: /images/mongosh-plp/Technical_ACTION_SearchDocument_Thumbnail_BS_Mist.png
+         :alt: Abstract illustration showing CRUD and aggregation operations
+
+   .. tab:: Administer and Manage
+      :tabid: administer
+
+      .. procedure::
+
+         .. step:: Manage Databases and Collections
+
+            View information about databases, create collections or views,
+            or drop databases - all from your shell. See all 
+            :ref:`Database Methods <mongosh-database-methods>`.
+
+            Perform collection operations, create or delete indexes, or 
+            explain queries with :ref:`Collection Methods 
+            <mongosh-collection-methods>`.
+
+         .. step:: Administer Servers
+
+            Manage :ref:`replication <mongosh-replication-methods>` or 
+            :ref:`sharding <mongosh-sharding-methods>` from the comfort 
+            of your shell.
```

Comment:
Suggestion: this sounded a little odd to me, what do you think of this language instead?
```suggestion
            :ref:`sharding <mongosh-sharding-methods>` conveniently in
            your shell.
```